css/style.css

```diff
@@ -10,6 +10,7 @@ ul {
 li {
     width: 25%;
     color: aqua;
+    background-color: bisque;
```

Comment:
> Please use the background color that's mentioned in your project.

- [ ] I would implore you to explain the reason for your comment so that the student can understand and think along your thought.

Reply:
- [Optionnal] Kindly uses a couple of emojis and/or gifs in the review. They make the code review more human-friendly 👽🤖.